vector/src/main/java/im/vector/app/features/spaces/manage/SpaceSettingsController.kt

```diff
@@ -133,13 +134,24 @@ class SpaceSettingsController @Inject constructor(
                 id = "manage_rooms",
                 title = stringProvider.getString(R.string.space_settings_manage_rooms),
                 // subtitle = data.getJoinRuleWording(stringProvider),
-                divider = vectorPreferences.developerMode(),
+                divider = true,
```

Comment:
`vectorPreferences.developerMode() || isPublic` ?

Reply:
it came from the rebase. Why are we using vectorPreferences.developerMode() to show hide dividers?

Reply:
It's because the next item is added only if `vectorPreferences.developerMode()` is true, and we do not add divider after the last item.

Reply:
done